client/extensions/woocommerce/app/settings/payments/payment-method-item.js

```diff
@@ -113,12 +150,21 @@ class PaymentMethodItem extends Component {
 		);
 	}
 
+	renderEnabledField = ( isEnabled ) => {
+		return (
+			<PaymentMethodEditFormToggle
```

Comment:
The CompactFormToggle does not emit an event.  It will work here but not everywhere else.  Here I know what toggle was clicked, in the settings cards I don't know what toggle was clicked since multiple toggles use the same handler